pytest_client_tools/insights_client.py

```diff
@@ -237,6 +237,20 @@ def core_version(self):
         assert m
         return Version(m.group(1))
 
+    @property
+    def client_version(self):
```

Comment:
since this is the `InsightsClient` class, the "client" bit in the name is a bit redundant; simply call this "version"